src/style.css

```
@@ -0,0 +1,103 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  --header__height: 60px;
+  --item__size: 15px;
+  --line__size: 4px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  min-height: var(--header__height);
+  align-items: center;
+  /*background-color: gray;*/
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  border-radius: 8px;
+}
+
+.header:focus,
+.header:hover {
+  outline: 1px solid #00acdc;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  align-self: stretch;
+  margin-right: 50px;
+  /*background-color: yellow;*/
+}
+
+.menu { 
+  /*position: relative;*/
+  min-height: var(--header__height);
+  margin-top: 0;
+  margin-bottom: 0;
```

```suggestion
  margin: 0;
```
Let's reset all margins, not only horizontal

Also, you should reset padding for the list (`padding: 0`) because it has padding by default and you don't need this